linux_os/guide/services/dhcp/disabling_dhcp_server/package_dhcp_removed/rule.yml

```diff
@@ -56,6 +56,7 @@ template:
         pkgname: dhcp
         pkgname@rhel8: dhcp-server
         pkgname@rhel9: dhcp-server
+        pkgname@rhel10: dhcp-server
```

Comment:
This package doesn't exist in RHEL 10 so this rule should be removed from RHEL 10.